### DISCUSSION TITLE
Rust Debugging - CodeLLDB

### DISCUSSION THREAD

**Discussion by nicraMarcin** [zed-industries/zed#19286] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Hi, 
Is any solution, plans to add CodeLLDB or something similar/better to debug code?
If yes, how can I implement this?

**Comment by SomeoneToIgnore** [comment]
There are issues to track: https://github.com/zed-industries/zed/issues/5307 and https://github.com/zed-industries/zed/issues/5065

The implementation is ongoing: https://github.com/zed-industries/zed/pull/13433
Not sure if any help is needed, though.

**Reply by nicraMarcin** [reply]
Super, so when can we expect it in release?

**Reply by SomeoneToIgnore** [reply]
When the PR is merged, no earlier.
No clue when that happens.

**Comment by Linyuqiz** [comment]
ok, I'm looking forward
